src/deprecated.jl

```diff
@@ -1,8 +1,7 @@
-const ZPKFilter = ZeroPoleGain
-const TFFilter = PolynomialRatio
-const BiquadFilter = Biquad
-const SOSFilter = SecondOrderSections
-export ZPKFilter, TFFilter, BiquadFilter, SOSFilter
+# deprecations after 0.5
+Base.@deprecate_binding ZPKFilter ZeroPoleGain
```

Comment:
What's the difference between `@deprecate` and `@deprecate_binding`?

Reply:
`@deprecate` only works for functions. I.e. after `@deprecate ZPKFilter ZeroPoleGain`, one could call `ZPKFilter(z, p, k)`, but could not use it as a type as in `f::ZPKFilter`.

Reply:
Ahh ok